### ISSUE TITLE
Referer policies don't work

### ISSUE THREAD

**Issue by nodomain** [frux/csp-header#32]
Can you please show an example how to add referer policies to the CSP? I don't get it to work.
```javascript
// referer values
export const NO_REFERER = "'no-referer'";
export const NONE_WHEN_DOWNGRADE = "'none-when-downgrade'";
export const ORIGIN = "'origin'";
export const ORIGIN_WHEN_CROSS_ORIGIN = "'origin-when-cross-origin'";
export const UNSAFE_URL = "'unsafe-url'";
```

Thanks

**Comment by frux** [comment]
Hi! It supposed to be used like this
```js
const { getCSP, NO_REFERER } = require('csp-header');

getCSP({
    directives: {
        referrer: [NO_REFERER]
    }
});
```

But I've just noticed that there is a typo in the value of `NO_REFERER`. It should be `'no-referrer'` (with double r). That's why it doesn't work.

Workaround is to not use constant
```js

getCSP({
    directives: {
        referrer: ["'no-referrer'"]
    }
});
```

I'm gonna prepare a patch for that. Thank you!

**Comment by frux** [comment]
I've just released patch in version 2.1.2
